contracts/scripts/deployRound.ts

```diff
@@ -0,0 +1,225 @@
+  let userRegistry: Contract
+  if (userRegistryType === 'simple') {
+    const SimpleUserRegistry = await ethers.getContractFactory(
+      'SimpleUserRegistry',
+      deployer
+    )
+    userRegistry = await SimpleUserRegistry.deploy()
+  } else if (userRegistryType === 'brightid') {
+    const BrightIdUserRegistry = await ethers.getContractFactory(
+      'BrightIdUserRegistry',
+      deployer
+    )
+    // TODO sort out if these arguments are sensible
+    userRegistry = await BrightIdUserRegistry.deploy(
+      '0x636c722e66756e64000000000000000000000000000000000000000000000000', // Do we have a test `context`?
+      fundingRoundFactory.address // Not sure if `verifier` should be EOA or contract account
+    )
+  } else {
+    throw new Error('unsupported user registry type')
+  }
+  await userRegistry.deployTransaction.wait()
+  console.log('userRegistry.address: ', userRegistry.address)
+
+  const setUserRegistryTx = await fundingRoundFactory.setUserRegistry(
+    userRegistry.address
+  )
+  await setUserRegistryTx.wait()
+
+  const recipientRegistryType = process.env.RECIPIENT_REGISTRY_TYPE || 'simple'
+  let recipientRegistry: Contract
+  if (recipientRegistryType === 'simple') {
+    const SimpleRecipientRegistry = await ethers.getContractFactory(
+      'SimpleRecipientRegistry',
+      deployer
+    )
+    recipientRegistry = await SimpleRecipientRegistry.deploy(
+      fundingRoundFactory.address
+    )
+  } else if (recipientRegistryType === 'optimistic') {
+    const OptimisticRecipientRegistry = await ethers.getContractFactory(
+      'OptimisticRecipientRegistry',
+      deployer
+    )
+    recipientRegistry = await OptimisticRecipientRegistry.deploy(
+      ethers.BigNumber.from(10).pow(ethers.BigNumber.from(18)).div(10),
+      300,
+      fundingRoundFactory.address
+    )
+  } else {
+    throw new Error('unsupported recipient registry type')
+  }
+  await recipientRegistry.deployTransaction.wait()
+  console.log('recipientRegistry.address: ', recipientRegistry.address)
+
+  const setRecipientRegistryTx = await fundingRoundFactory.setRecipientRegistry(
+    recipientRegistry.address
+  )
+  await setRecipientRegistryTx.wait()
+
+  const setTokenTx = await fundingRoundFactory.setToken(
+    process.env.NATIVE_TOKEN_ADDRESS
+  )
+  await setTokenTx.wait()
+
+  // Generate coordinator key
+  const keypair = new Keypair()
+  const coordinatorPubKey = keypair.pubKey
+  const serializedCoordinatorPrivKey = keypair.privKey.serialize()
+  const serializedCoordinatorPubKey = keypair.pubKey.serialize()
+  const setCoordinatorTx = await fundingRoundFactory.setCoordinator(
+    deployer.address,
+    coordinatorPubKey.asContractParam()
+  )
+  await setCoordinatorTx.wait()
+  console.log('serializedCoordinatorPrivKey: ', serializedCoordinatorPrivKey)
+  console.log('serializedCoordinatorPubKey: ', serializedCoordinatorPubKey)
+  console.log('coordinatorPubKey.serialize(): ', coordinatorPubKey.serialize())
```

Comment:
These will always be the same... we could keep the `serializedCoordinatorPubKey` for variable readability and remove `console.log('coordinatorPubKey.serialize(): ', coordinatorPubKey.serialize())`
...or we could remove both:
```js
const serializedCoordinatorPubKey = keypair.pubKey.serialize() // L101

console.log('serializedCoordinatorPubKey: ', serializedCoordinatorPubKey) // L108
```

(would update the `admin.md` "Info to keep track of" as well)